build/module-defaults.mk

```diff
@@ -37,7 +37,7 @@ ifeq ($(USE_SWD_JTAG),y)
 COMPILE_LTO ?= n
 endif
 
-WARNINGS_AS_ERRORS ?= y
+WARNINGS_AS_ERRORS ?= n
```

Comment:
This should be left as-is. Only the application is allowed to be built with just warnings.